autovalet_lane_detection/scripts/autovalet_lane_detection/autovalet_lane_detector.py

```diff
@@ -68,7 +73,7 @@ def registered_image_callback(self, color_msg, depth_msg):
     rospy.init_node('lane_detector_node')
 
     color_topic     = "/frontCamera/color/image_raw"
-    depth_topic     = "/depth_registered/image_rect"
+    depth_topic     = "/frontCamera/aligned_depth_to_color/image_raw"
```

Comment:
<another big change> Sure this topic would be consistent between he and sim? I remember there was some ambiguity there

Reply:
no the topics are different between hardware and sim, but here it's merely the code under __main__ which we shouldn't be activating in the overall system execution. Basically, I think this is only for debugging and leaving it the way it is should be fine?

Reply:
i thought we specified which topics to use in launchfiles, not in code?

Reply:
yes we need. that's why the code here is only under __main__ which is for unit testing.

Reply:
Got it. My bad. Missed that this main.